.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: false
+git:
+  depth: 3
+language: java
+env:
+  global:
+  - TERM=dumb         # for gradle
+matrix:
+  include:
+  - name: 'Java 8'
+    jdk: oraclejdk8
+
+install: true
+script:
+- (cd jib-core; ./gradlew --stacktrace build)
+- (cd jib-plugins-common; ./gradlew --stacktrace build)
+- (cd jib-gradle-plugin; ./gradlew --stacktrace build)
+# unset _JAVA_OPTIONS to avoid spurious test failures
+- (cd jib-maven-plugin; unset _JAVA_OPTIONS; ./mvnw -B verify -U)
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
```

Comment:
Curious: what is this `plugin-resolution` directory for?

Reply:
looks like there some other lock in there that shouldn't be preserved across builds?

Reply:
Oh jeepers, what a mess.

Reply:
Sorry, I meant to say, I agree with you.